framework/core/device.cpp

```diff
@@ -253,7 +227,7 @@ Device::Device(const PhysicalDevice &gpu, VkSurfaceKHR surface, std::unordered_m
 
 	command_pool = std::make_unique<CommandPool>(*this, get_queue_by_flags(VK_QUEUE_GRAPHICS_BIT | VK_QUEUE_COMPUTE_BIT, 0).get_family_index());
 	fence_pool   = std::make_unique<FencePool>(*this);
-}
+}        // namespace vkb
```

Comment:
was this accidentally added by clang-format?

Reply:
If I remove it and clang-format the file it comes back. So deliberate by clang-format I guess.

Reply:
yes unfortunately we had this issue before, annoying because that bracket doesn't close the namespace 😞

Reply:
Are you sure? It looks like the namespace closure to me

Reply:
that's the one in line 622 that was there before, but this one is in line 230

Reply:
Ah yes - hadn't noticed the line number wasn't the end of the file.
That one does go away, so clang-format must have added it when I had some mismatched braces during development I guess.

I'll push a fix. Thanks

Reply:
Done